dll/win32/shell32/CDefView.cpp

```diff
@@ -3008,6 +3008,9 @@ LRESULT CDefView::OnSettingChange(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL
     if (wParam == SPI_SETDESKWALLPAPER || wParam == 0)
         UpdateListColors();
 
+    if (wParam == SPI_SETICONTITLELOGFONT || !wParam)
```

Comment:
```suggestion
    if (wParam == SPI_SETICONTITLELOGFONT || wParam == 0)
```
if you want to keep the same style as above.